docs/pages/database-access/guides/rds.mdx

```diff
@@ -187,6 +187,10 @@ $ tsh db ls
   `<cluster-id>-custom-<endpoint-name>` respectively.
 </Admonition>
 
+<Admonition type="note" title="Note">
```

Comment:
```suggestion
```
I've suggested combining the two Admonitions since the second one works well as the second paragraph in a single Admonition.